pkg/handlers/generic/lifecycle/config/cm.go

```diff
@@ -76,18 +76,18 @@ func (h *HelmChartGetter) For(
 	name Component,
 ) (*HelmChart, error) {
 	log.Info(
-		fmt.Sprintf("Fetching HelmChart info for %s from configmap %s/%s",
+		fmt.Sprintf("Fetching HelmChart info for %q from configmap %s/%s",
 			string(name),
-			h.cmName,
-			h.cmNamespace),
+			h.cmNamespace,
+			h.cmName),
```

Comment:
Really nitty but interesting all the same imo: I'm starting to prefer creating `client.ObjectKey` and relying on the stringifying of that rather than relying on ordering like this - I've seen it mixed up so many times. Also no need to cast `name` to a string.

```suggestion
		fmt.Sprintf(
			"Fetching HelmChart info for %q from configmap %s",
			name,
			client.ObectKey{Namespace: h.cmNamespace, h.cmName},
		),
```